src/libs/PusherConnectionManager.js

```diff
@@ -16,16 +17,20 @@ function init() {
     }));
 
     /**
-     * Events that happen on the pusher socket are used to determine if the app is online or offline.
-     * The offline setting is stored in Onyx so the rest of the app has access to it.
-     *
```

Comment:
Removing this as it is not accurate. The events are not used to determine online/offline status.